docs_src/OVMS/supportingDifferentLanguage.md

```diff
@@ -1,15 +1,17 @@
-For running OVMS as inferencing engine through grpc, we are supporting multiple programming languages for your need. Here is the list of languages we are supporting:
+# Supporting Different Languages
+
+For running OVMS as inferencing engine through grpc, these are the supported languages:
 
 1. python
 2. golang
 
-Here is the script example to start grpc-python using rtsp as inputsrc:
-`PIPELINE_PROFILE="grpc_python" sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0`
-
-Here is the script example to start grpc-go using rtsp as inputsrc:
-`PIPELINE_PROFILE="grpc_go" sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0`
+## Sample Commands using the Camera Simulator
+| Input source Type |Command                                                                                                                                        |          
+|-------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
+| Python | `PIPELINE_PROFILE="grpc_python" sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0` |
+| Golang | `PIPELINE_PROFILE="grpc_go" sudo -E ./run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0` |
```

Comment:
since camera_0 is meant for barcode detection, maybe we put camera_1 here for the example